### PR TITLE
Incorporating Jenn's comments on search page text

### PR DESCRIPTION
**Pull request by taceybadgerbrook**
comments as per https://github.com/wellcomecollection/wellcomecollection.org/issues/5386

### CODE REVIEW

File: docs/search/README.md [hunk 1]
```diff
@@ -2,11 +2,7 @@
 
 ## Search relevance
 
-Our current setup for enabling discovery of material on wellcomecollection.org/works is pretty basic. Since going live, api.wellcomecollection.org has used the default elasticsearch `simple_query_string` query type, and we haven't made any significant index-time enhancements \(like indexing n-gram tokens\) to the underlying data.
-
-The single point of interaction \(the search box\) and often hundreds of pages of results to sift through leaves users frustrated, feeling a lack of control over the interface and a weakening trust in the system as a whole.
-
-### Baby steps
+When api.wellcomecollection.org went live, we used the default elasticsearch `simple_query_string` query type for enabling discovery of material on wellcomecollection.org/works.
```
Comment (jamesgorrie) [inline docs/search/README.md:5]: Could we add some links?

```suggestion
When [api.wellcomecollection.org](https://api.wellcomecollection.org) went live, we used the default [elasticsearch `simple_query_string`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html) query type for enabling discovery of material on [wellcomecollection.org/works](https://wellcomecollection.org/works).
```

File: docs/search/README.md [hunk 2]
```diff
@@ -20,15 +16,13 @@ We're taking a [data informed](https://stacks.wellcomecollection.org/data-inform
 
 In broad terms, we want to produce a number which tells us whether a candidate change to the search system makes the results more or less relevant to the person searching.
 
-There's a lot of interesting discussion to be had about different search modalities, precision vs serendipity, linked open data & knowledge graphs, and existential questions about what relevance even _is_, dude, but the broad definition above is a useful, concise statement to revert back to.
```
Comment (jamesgorrie) [inline docs/search/README.md:23]: Haha - I wonder how this got here.

File: docs/search/README.md [hunk 3]
```diff
@@ -20,15 +16,13 @@ We're taking a [data informed](https://stacks.wellcomecollection.org/data-inform
...
+There's a lot of interesting discussion to be had about different search modalities, precision vs serendipity, linked open data & knowledge graphs, and existential questions about what relevance even _is_ but the broad definition above is a useful, concise statement to revert back to.
 
 #### Explicit vs implicit measurement
 
 We're collecting data in two distinct ways:
 
-* **Implicit data collection:** users know that their data might be used to understand aggregate user behaviour, but their behaviour on the site is driven by another motive.
-
-The larger of the two major datasets we are using is passively collected while users carry out searches and look through pages of results.
+* **Implicit data collection:** users know that their data might be used to understand aggregate user behaviour, but their behaviour on the site is driven by user search requirements and paging through results.
```
Comment (jamesgorrie) [inline docs/search/README.md:25]: I'm not sure I understand, a person's behaviour is driven by our research and paging? How are we driving their behaviour?

File: docs/search/README.md [hunk 4]
```diff
@@ -43,13 +37,9 @@ We can collect lots of this data \(100,000s of actions logged so far\), but it l
...
+Some of the data we've used to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that people / researchers might be looking for, following a set of instructions to obtain data in a useful format. This data is small \(20-50 rows per person, per session, with ~10 people taking part\), but can be deeply contextualised by talking to the people who took part.
 
-The resulting data matches the format of the small-scale research described above, but can be collected at a much faster pace with less use of an analyst's time. The increased sample size and breadth of subjects evaluated should also give us more confidence in the 'goodness' of the changes we're making to our search algorithms.
+We've also collected data by diary studies, asking users to provide their real life search queries and apply a ranking to them and by having an opt-in relevance ranker, allowing those who wanted to rank search results could do so.
```
Comment (jamesgorrie) [inline docs/search/README.md:42]: Not sure the what the need for "opt-in relevance ranker" is here, as it won't make sense outside of Wellcome Collection without an example.

File: docs/search/README.md (hunk 1 above)
Comment (taceybadgerbrook) [reply docs/search/README.md:5]: Good idea.

File: docs/search/README.md (hunk 2 above)
Comment (taceybadgerbrook) [reply docs/search/README.md:23]: Poetic licence?

File: docs/search/README.md (hunk 3 above)
Comment (taceybadgerbrook) [reply docs/search/README.md:25]: Hmm, maybe better, "their behaviour on the site is driven by their own research requirements."

File: docs/search/README.md (hunk 4 above)
Comment (taceybadgerbrook) [reply docs/search/README.md:42]: Jenn suggested this be added as one of the many ways we've gathered user feedback. Could change text to read, "staff-only opt-in relevance ranker, allowing those with in-depth knowledge of the Collection to rank search results."

File: docs/search/README.md (hunk 2 above)
Comment (jamesgorrie) [reply docs/search/README.md:23]: Maybe we should try write all docs in rhyming couplets?

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**